.github/workflows/types-integration.yml

```diff
@@ -146,3 +146,24 @@ jobs:
       - name: Run TSC
         working-directory: eslint-js
         run: npm run build:types --workspace eslint-visitor-keys
+
+  are-the-types-wrong:
+    name: Are the types wrong?
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout eslint
+        uses: actions/checkout@v4
+        with:
+          path: eslint
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: "lts/*"
+
+      - name: Install Packages
+        working-directory: eslint
+        run: npm install
+
+      - name: Check validity of type definitions
+        working-directory: eslint
+        run: npx @arethetypeswrong/cli@latest -P
```

Comment:
Running `@latest` does make me a little nervous 😬 ... you never know what breaking changes happen across new major versions. I don't know what the TSC's preference would be, and I don't have a strong one myself. Asking to future reviewers: would it be preferable to have this be pinned?

Reply:
Makes sense to pin the npm dependency to the current major version 👍

Reply:
Sounds good.